### ISSUE TITLE
Batch effect correction

### ISSUE THREAD

**Issue by zhenzuo2** [KChen-lab/marker-selection#2]
Hi,

Thank you so much for sharing this tool. I am not sure how to deal batch effect before running this pipeline? Thank you so much.

Best,

Zhen

**Comment by lshh125** [comment]
Thank you for your interest! You can refer to this example: https://scmer.readthedocs.io/en/latest/ileum-lamina-propria-immunocytes.html

In block 20, we used model.fit(adata.X, batches=adata.obs['batch'].values), where the "batches" is the label for samples. So if you have 3 samples, and 1,000 per sample, you would have something like batches = [0] * 1000 + [1] * 1000 + [2] * 1000.

Does this answer your question?

**Comment by lshh125** [comment]
I'll close the question now. If you have additional questions few free to reopen it. Thanks.
